packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
             className={ css('ms-DatePicker-table', styles.table) }
-            role='grid'
+            role='application'
```

Comment:
Table structure is not useful in case of calendar, as we don't want to announce column, column headers and rows given its intuitive in case of calendar. I am following the one provided at http://whatsock.com/tsg/Coding%20Arena/ARIA%20Date%20Pickers/ARIA%20Date%20Picker%20%28Basic%29/demo.htm. It aligns with our windows Calendar as well.